src/pages/index.astro

```diff
@@ -8,9 +8,10 @@ import DevToken from '@components/Home/Heros/DevToken.astro'
 
 import Backers from '@components/Home/Sections/Backers.astro'
 import Blog from '@components/Home/Sections/Blog.astro'
-import Dapps from '@components/Home/Sections/Dapps.astro'
+// import Dapps from '@components/Home/Sections/Dapps.astro'
```

Comment:
Humm this can be removed.

Reply:
hmm true. sorry forgot to do that 😅

Reply:
Removed in https://github.com/web3community/devprotocol.xyz/commit/dd7e5d0e2b8289244a17ac348d043477a8897627